pages/news-post.html

```diff
@@ -0,0 +1,211 @@
+    </header>
+    <!-- Search Bar Overlay, hidden initially. -->
+    <section
+      id="search-drawer-from-header"
+      class="fixed border border-black inset-x-0 top-0 w-full h-16 bg-white hidden transform -translate-y-full transition-transform duration-500 ease-out z-50"
+      aria-label="search-bar"
+    >
+      <h2 id="search-label" class="sr-only">Search Bar</h2>
+      <div class="flex justify-center items-center pt-3">
+        <div
+          class="w-full max-w-lg mx-auto border border-black bg-gray-100 flex items-center"
+        >
+          <form class="flex flex-grow">
+            <label for="Search-Drawer" class="sr-only">Search Site</label>
+            <input
+              type="search"
+              class="search-drawer_input flex-grow bg-transparent focus:outline-none p-2 text-sm"
+              id="Search-Drawer"
+              placeholder="Search..."
+              autocomplete="off"
+            />
+            <button
+              type="submit"
+              class="search-drawer_submit p-2 text-gray-500 flex items-center justify-center"
+            >
+              <img
+                src="../images/SVGs/Search.svg"
+                class="w-4 h-4"
+                aria-label="Search Now Button"
+                alt="Search Now Button"
+              />
+            </button>
+          </form>
+        </div>
+        <!-- Close button ready for JS. Should also close on outside-click, not just by close button itself -->
+        <button
+          id="closeSearchBarDrawer"
+          class="search-drawer_close text-sm pr-5"
+        >
+          Close
+        </button>
+        <!-- JS intention: On click, add -translate-y-full to section id="search-drawer-from-header" and remove translate-y-0 -->
+      </div>
+    </section>
+    <!-- Section 3 -->
+    <!-- Support us -->
+<section class="flex flex-col gap-20">
+    <div class="flex flex-col justify-center mx-[169px] my-5 max-w-[1200px] bg-ivory-beige items-center pt-16 pb-10 px-[200px] rounded-[5px] shadow-[10px_10px_3px] shadow-sunset-red"
+      >
+        <button class="rounded-lg alt-button text-center px-5 py-4 mx-10 text-6xl font-normal tracking-wider mb-8">
+          Support us
+        </button>
+        <p class="text-lg my-2">
+          The funds that come in go towards covering everything from food,
+          clothing, education, equipment, healthcare, medicines, and dental care
+          for the children, as well as the operation of the music school and
+          salaries for our staff.
+          <p class="text-lg my-2">The families of the children are also included
+          in the food program and receive monthly distributions of food and
+          hygiene products.</p>
+          <p class="text-lg my-2">We are constantly working to improve the living
+          conditions of the children, and we use much of the surplus we have to
+          repair houses and homes.</p>
+        </p>
+      </div>
+
+    <!-- Newsletter -->
```

Comment:
Should Newsletter be a separate <section>? When I look at the design file it looks like they could be two different sections.  Now Newsletter and Support us is placed inside the same section. If this is intentional it's fine :)

Reply:
I just looked at the updated design file, that was different from the preview picture in the issue. So I intentionally just added the newsletter div into the same section. There haven't been opened any issues for a fourth section of the page containing the newsletter content, so I think it's alright to have it in the same section. But I can ofc update with two different sections.

Reply:
Yo vic, i see Anne has split the newsletter into a separate section in the updated agency theme dev file, so lets just do that.

Section 4 = newsletter

Reply:
Great! I'll wait on that change :)

Reply:
Perfect! Thank you!